editor/src/components/canvas/design-panel-root.tsx

```diff
@@ -301,21 +309,50 @@ export const DesignPanelRoot = betterReactMemo('DesignPanelRoot', (props: Design
             >
               <TopMenu />
             </SimpleFlexRow>
+
+            {isCanvasVisible && props.isUiJsFileOpen && navigatorPosition !== 'hidden' ? (
+              <div
+                style={{
+                  height: `calc(100% - ${TopMenuHeight}px)`,
+                  position: 'absolute',
+                  top: TopMenuHeight,
+                  left: 0,
+                  zIndex: 20,
+                  overflow: 'hidden',
+                }}
+              >
+                <ResizableFlexColumn
+                  style={{
+                    overscrollBehavior: 'contain',
+                    backgroundColor: UtopiaTheme.color.bg0.o(90).value,
+                    backdropFilter: 'blur(7px)',
+                  }}
+                  defaultSize={{
+                    width: 280,
```

Comment:
We have `LeftPaneDefaultWidth` for this, which is being used elsewhere in the editor for some offsetting. I'm not sure if you want to update that to 280 and use it here

Reply:
Good question. I think I'll keep them separate for now, not sure if the ideal navigator width is the same thing as the ideal leftMenu width.